frontend/packages/ux-editor/src/App.tsx

```diff
@@ -75,6 +79,16 @@ export function App() {
     }
   }, [setSelectedLayoutSet, selectedLayoutSet, layoutSets, app]);
 
+  if (
+    isAppVersionFetched &&
+    appVersion.frontendVersion.startsWith('4') &&
+    !shouldDisplayFeature('shouldOverrideAppFrontendCheck')
+  ) {
```

Comment:
We can skip using `isSuccess` in the query and just use `data: appVersion` directly. When the query hasn't run yet, the data is undefined. Even if the query fails to fetch data, it stays undefined, but the error message in the query will have info on what went wrong. 😊 


```suggestion
  if (
    appVersion &&
    appVersion.frontendVersion.startsWith('4') &&
    !shouldDisplayFeature('shouldOverrideAppFrontendCheck')
  ) {
```

Reply:
I updated it 😊 Out of curiosity, what is the problem with using `isSuccess`?

Reply:
Nothing more than there is one less state to maintain/read. 🙈